mosaic-runtime/src/commonMain/kotlin/com/jakewharton/mosaic/ui/Box.kt

```diff
@@ -29,9 +42,54 @@ internal class BoxMeasurePolicy : MeasurePolicy {
 			}
 		}
 		return layout(width, height) {
-			for (placeable in placeables) {
-				placeable.place(0, 0)
+			placeables.forEachIndexed { index, placeable ->
+				val alignment = measurables[index].boxParentData?.alignment ?: contentAlignment
+				val offset =
+					alignment.align(IntSize(placeable.width, placeable.height), IntSize(width, height))
+				placeable.place(offset.x, offset.y)
 			}
 		}
 	}
 }
+
+/**
+ * A BoxScope provides a scope for the children of [Box].
+ */
+@LayoutScopeMarker
+@Immutable
+public interface BoxScope {
+	/**
+	 * Pull the content element to a specific [Alignment] within the [Box]. This alignment will
+	 * have priority over the [Box]'s `alignment` parameter.
+	 */
+	@Stable
+	public fun Modifier.align(alignment: Alignment): Modifier
+}
+
+private object BoxScopeInstance : BoxScope {
+
+	@Stable
+	override fun Modifier.align(alignment: Alignment) = this.then(
+		AlignModifier(alignment = alignment)
+	)
+}
+
+private class AlignModifier(
+	private val alignment: Alignment
```

Comment:
```suggestion
	private val alignment: Alignment,
```
Really gotta get ktlint going on this repo...